nodejs/week3/src/backend/api/meals.js

```diff
@@ -0,0 +1,148 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+
+//Get all meals
+router.get("/", async (req, res) => {
+  try {
+    const mealslist = await knex("meals").select("*");
+    res.send(mealslist);
+  } catch (error) {
+    throw error;
+  }
+});
+
+router.get("/titles", async (request, response) => {
```

Comment:
this route is not part of the exercise. These are the routes that need to be implemented:


api/meals/
api/meals/:id
api/reservations/
api/reservations/:id
api/reviews/
api/reviews/:id

Reply:
yap,.,i will remove it, tak